developer/js/source/lexical-model-compiler/lexical-model-compiler.ts

```diff
@@ -49,17 +50,30 @@ export default class LexicalModelCompiler {
         // file, rather than the current working directory.
         let filenames = modelSource.sources.map(filename => path.join(sourcePath, filename));
 
-        // Use the default search term to key function, if left unspecified.
-        let searchTermToKey = modelSource.searchTermToKey || defaultSearchTermToKey;
+        let pseudoclosure = new ModelPseudoclosure(modelSource);
+        
+        func += pseudoclosure.compilePseudoclosure();
```

Comment:
Instead of the name `ModelPseudoclosure` perhaps use the name `ModelDefinitions`, as this effectively adds the definitions of `applyCasing`, `searchTermToKey`, and any defaults when needed, e.g.,

```suggestion
        let definitinos = new ModelDefinitions(modelSource);
        
        func += definitions.compileDefinitions();
```

I feel like the term "pseudoclosure" is needlessly complicated 😅